src/Akeneo/Platform/Job/back/Application/FindJobTypes/FindJobTypesHandler.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Platform\Job\Application\FindJobTypes;
+
+/**
+ * @author Pierre Jolly <pierre.jolly@akeneo.com>
+ * @copyright 2021 Akeneo SAS (https://www.akeneo.com)
+ * @license https://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+final class FindJobTypesHandler
```

Comment:
Same for the interface
```suggestion
final class FindJobTypeHandler
```